schema.ts

```diff
@@ -1201,7 +1211,21 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
    * @hidden
    */
   modifiedByProperty?: PropertyIdentifier<K>;
-
+  /**
+   * The name of a property within {@link ObjectSchemaDefinition.properties} that can be interpreted as the email
```

Comment:
Probably `For cases when the object being synced represents a user, the name of the property within {@link ObjectSchemaDefinition.properties} that identifies the email address of the user.`